outlines/1.3-migration

```diff
@@ -0,0 +1,31 @@
+1. Breaking changes (as we'd normally do this)
+
+2. Recommended changes : Modules [link to app structure guide]
+  1. Applications: Move all files to imports, initialize in one spot, link via `import`
+  2. Import *all* symbols from packages, including meteor core packages.
```

Comment:
It would be unfortunate though if enabling `import { loginWithPassword } from 'meteor/accounts-password';` necessarily meant that you would always get `window.loginWithPassword` in all of your files. It would be wonderful if it were possible to _reduce_ the number of globals rather than increase...